kernel/terminal.cpp

```diff
@@ -468,13 +447,21 @@ void Terminal::ExecuteLine() {
     (*layer_task_map)[layer_id_] = subtask_id;
   }
 
-  if (strcmp(command, "echo") == 0) {
-    if (first_arg && first_arg[0] == '$') {
-      if (strcmp(&first_arg[1], "?") == 0) {
+  if (strcmp(command, ">") == 0) {
+    PrintToFD(*files_[2], "no command befoer > \n");
```

Comment:
befoer は before の typo

Reply:
Bash だと下記のコードは動きますが、MikanOS ではエラーにするんですね。

```
$ > a echo because
$ cat a
```

リダイレクトの位置について、何らかの暗黙的な制約がある（リダイレクトは一番最後に書かないといけない？）
ように思われるので、その制約を関数ドキュメントに明記してください。

Reply:
https://github.com/uchan-nos/mikanos/pull/64/commits/a75639226c9a837717b383dea0deeb3e7ffdb7d4
typoを修正しました

Reply:
リダイレクトの位置についての制約はどの関数の関数ドキュメントに書けばいいですか？

Reply:
リダイレクトの位置の制約は ExecuteLine の関数ドキュメントにお願いします

Reply:
https://github.com/uchan-nos/mikanos/commit/cf37d706c849d2cb928bc473e945c95e68ff9f75
リダイレクトの位置の制約について ExecuteLine の関数ドキュメントに書きました